torchtitan/models/llama_multimodal/model.py

```diff
@@ -9,19 +9,23 @@
 
 import math
 from dataclasses import dataclass
-from typing import Any, List, Optional, Tuple
+from typing import Any, Dict, List, Optional, Tuple
 
 import torch
 import torch.nn as nn
 import torch.nn.functional as F
 
+from torchtitan.models.norms import RMSNorm
+
 
 @dataclass
 class ModelArgs:
```

Comment:
Since encoder and decoder share `ModelArgs`, let's have three sections: shared, encoder-only, decoder-only.

Reply:
Hmmm after a second thought this might not be a good idea. Because hugging face explicitly oppose inheritance. Or we are saying we don't want to go with the hugging face way?

Reply:
I'm not saying we should have three classes. I mean we should visually separate the variables in this class, using spaces and comments. O/w one cannot easily tell if a variable is encoder-only, decoder-only, or shared.

Reply:
sure.. I can make it different sections.